m2-q1.txt

```diff
@@ -0,0 +1,41 @@
+js fundamental variable assignment
+
+let a ;
+console.log("before assignment : "+a)
+a=10
+console.log("after assignment : "+a)
+
+if we log before assigning any value we get undefined after assigning we get the assigned value.
+we get same corresponding output for var but when we use const an error is generated 'Missing initializer in const declaration'.
+
+
+
+For any numeric value wether it be integer or decimal we get the datatype 'number' and respectively for string we get 'string'.
+For array and null typeof returns object and for NaN it gives number.
+
+
+
+
+Other way to check whether its an array or not is using Array.isArray() method.
```

Comment:
With textual answer also add the code or actual examples which you have executed